clients/fidesui/README.md

```diff
@@ -0,0 +1,21 @@
+# Fides UI
+
+This package is a result of migrating the [ethyca/fidesui](https://github.com/ethyca/fidesui) standalone repo into this monorepo. It includes components that are custom built for FidesUI. They may be built from scratch, or use a variety of Chakra components to create something new.
+
+## Migration notes
+
+A few of the main differences between the standalone and this are:
+
+- This is now part of the turbo monorepo when imported as an [Internal Package](https://github.com/vercel/turbo/blob/v1.9.3/docs/pages/repo/docs/handbook/sharing-code/internal-packages.mdx)
+- Fides UI now gets bundled by NextJS using the [next-transpile-modules](https://www.npmjs.com/package/next-transpile-modules) npm package
+- For now does not have a Storybook implementation as we investigate and determine the future of this project.
+- In light of all of the above, Lerna is no longer needed or included.
+
+## What should go in here
+
+In general, components that need to be shared across Fides applications should go in here in order to ensure a consistent UI. This may involve needing to import types specific to the Fides API as needed.
```

Comment:
Can you write something briefly re: how we dev against fidesui? E.g. when we make a change here can we test directly in `clients/fidesui` or are we limited to testing full-stack using admin-ui?

Reply:
great question and suggestion. I'm about to add linting to it, as well as type checking. Testing has always been in the admin-ui and fides-ui projects, I'll pull over some of the language from the original repo about that here as well.